src/app/[locale]/class-based/(classwork)/components/common/details/ContentSection.tsx

```
@@ -43,22 +45,40 @@ type Props = BaseProps & (WithGradeProps | WithoutGradeProps) & (WithDeadlinePro
+                            </div>
+                        )}
+                        <div className="flex items-center gap-1.5">
+                            <span className="font-medium">{tCommon('labels.createdAt')}:</span>
+                            <span>{formatDate(createdAt)}</span>
+                        </div>
+                    </div>
+                </div>
+                {description && (
+                    <div 
+                        className="text-sm text-muted-foreground break-words max-h-[400px] overflow-y-auto pr-2 prose prose-sm max-w-none dark:prose-invert"
+                        dangerouslySetInnerHTML={{ __html: renderMarkdown(description) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)